### ISSUE TITLE
Fix coloring on Changes in Eviction Filings chart bars when toggled to "filing counts"

### ISSUE THREAD

**Issue by james-minton** [EvictionLab/eviction-lab-website#656]
In the branch `bryony-ets-d3`, when Changes in Eviction Filings chart is toggled to "filing counts," all bars are red (image 1), but there should be red and blue bars instead (image 2). (The red and blue is what exists on the live site, but seems to have been overwritten by recent changes.)

<img width="683" alt="Screen Shot 2022-09-14 at 9 56 34 AM" src="https://user-images.githubusercontent.com/33635008/190175230-dd3a416b-1e1c-4b31-bde9-288c62b74953.png">
<img width="627" alt="Screen Shot 2022-09-14 at 9 57 04 AM" src="https://user-images.githubusercontent.com/33635008/190175246-aca91294-f84f-4e11-99f5-fc81b63259ad.png">

Also, the striped pattern should apply to both red and blue bars.

Further, when toggled to Filing Counts, the text "relative to average" should not appear in the legend. (It should still appear when toggled to "Filings relative to average," though.)

<img width="459" alt="Screen Shot 2022-09-14 at 9 57 26 AM" src="https://user-images.githubusercontent.com/33635008/190175824-c386837e-01b5-4211-8ae5-30b494f283ff.png">

In summary, when chart is in Filing Counts mode,

- [x] Change bars to red and blue 
- [x] Apply striped pattern to final 2 bars
- [x] Remove "relative to average", just when toggled to Filing Counts
